### PR TITLE
Avo Inspector in Web workers

### PR DESCRIPTION
**Pull request by alex-tpom6oh**

### CODE REVIEW

**Comment by hugihlynsson** [comment]
#### 📝 Changed routes:


| [**…tion/inspector/sdk/web**](https://docs-psqm8ooks.avo-dev.app/implementation/inspector/sdk/web) |
|:-:|
|<a href="https://docs-psqm8ooks.avo-dev.app/implementation/inspector/sdk/web">Screenshot of /implementation/inspector/sdk/web</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/O511q1FKmaR6MI6Ih7gizKJlF4yRNofJYSZPuU2IdTfvShTCZ0O5eXeOlCkiZKiZV75JpPHGlj1qVDBtyjyzvg.png">(view full size)</a> |







Commit 9bb6d26889590a7dfb86ce152e6375281267d46b (https://docs-psqm8ooks.avo-dev.app).

**Review by hugihlynsson — APPROVED** [review]

File: pages/implementation/inspector/sdk/web.mdx [hunk 1]
```diff
@@ -203,3 +203,11 @@ Enables manual control over events batching. The default production batch flush
 > #### Parameters:
 >
 > - `int newBatchFlushSeconds` - sets batch flush time in seconds.
+
+## <a name="web-workers"></a>Using in web workers
+
+The Web SDK maintains some shared state across the tracking calls and stores the data for batching, which makes it not suitable for web workers usage.
+
+If you have event tracking in Web Workers, we recommend to set up a listener in the main thread and send your event data there, where an inspector instance will be listening for it.
```
Comment (alex-tpom6oh) [inline pages/implementation/inspector/sdk/web.mdx:211]:
```suggestion
If you have event tracking in web workers, we recommend to set up a listener in the main thread and send your event data there, where an inspector instance will be listening for it.
```

Comment (alex-tpom6oh) [inline pages/implementation/inspector/sdk/web.mdx:211]:
```suggestion
If you have event tracking in Web Workers, we recommend to set up a listener in the main thread and send your event data there, where an inspector instance will be listening, processing and sending the event schemas to Avo.
```

Comment (alex-tpom6oh) [inline pages/implementation/inspector/sdk/web.mdx]:
```suggestion
If you have event tracking in Web Workers, we recommend to set up a listener in the main thread and send your event data there, where an inspector instance will be listening, processing the data and sending the event schemas to Avo.
```

File: pages/implementation/inspector/sdk/web.mdx [hunk 2]
```diff
@@ -203,3 +203,11 @@ Enables manual control over events batching. The default production batch flush
 > #### Parameters:
 >
 > - `int newBatchFlushSeconds` - sets batch flush time in seconds.
+
+## <a name="web-workers"></a>Using in web workers
+
+The Web SDK maintains some shared state across the tracking calls and stores the data for batching, which makes it not suitable for web workers usage.
+
+If you have event tracking in web workers, we recommend to set up a listener in the main thread and send your event data there, where an inspector instance will be listening for it.
```
Comment (alex-tpom6oh) [inline pages/implementation/inspector/sdk/web.mdx:211]:
```suggestion
If you have event tracking in web workers, we recommend to set up a listener in the main thread and send your event data there, where an inspector instance will be listening, processing and sending the event schemas to Avo.
```

**Pull request merged**